drivers/flash/Kconfig.stm32

```diff
@@ -29,13 +30,14 @@ config SOC_FLASH_STM32
 	select FLASH_HAS_PAGE_LAYOUT if SOC_SERIES_STM32F7X
 	select FLASH_HAS_PAGE_LAYOUT if SOC_SERIES_STM32L4X
 	select FLASH_HAS_PAGE_LAYOUT if SOC_SERIES_STM32WBX
+	select FLASH_HAS_PAGE_LAYOUT if SOC_SERIES_STM32WLX
 	select FLASH_HAS_PAGE_LAYOUT if SOC_SERIES_STM32G4X
 	select FLASH_HAS_PAGE_LAYOUT if SOC_SERIES_STM32H7X
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 	help
 	  Enable STM32F0x, STM32F1x, STM32F2x, STM32F3x, STM32F4x, STM32F7x,
-	  STM32L0x, STM32L1x, STM32L4x, STM32WBx, STM32G0x, STM32G4x or STM3H7x
-	  series flash driver.
+	  STM32L0x, STM32L1x, STM32L4x, STM32WBx, STM32WLx, STM32G0x, STM32G4x
+	  or STM3H7x series flash driver.
```

Comment:
I think this whole Kconfig section starts being confused and not clear enough so that users can be able to understand easily which series are supported  (and via which driver). We'll need to enhance this.
Anyway this is not blocking for current change

Reply:
Yeah the list in the description does not add much value at this point.